<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
  <head>
    <title>TEISER</title>
    <!--
<link rel="stylesheet" type="text/css" href="screen.css" media="screen" /> 
-->
<link rel="stylesheet" type="text/css" href="screen.css" />
<script type="text/javascript">

  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-31413568-1']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script>

  </head>

  <body>


<div id="page">
    <h1>
            <a href="http://www.columbia.edu/">
              <img src="css/university_logo.png" alt="Columbia University in the City of New York" />
            </a>
          </h1>
          <h1>
            <a href="http://www.tavazoielab.org/">
              <img src="css/lab_logo.png" alt="Tavazoie Lab" />
            </a>
          </h1>
          <div id="nav">
            <ul>
              <li>
              <a href="https://tavazoielab.c2b2.columbia.edu/lab/">Home</a>
              </li>
              <li>
              <a href="https://tavazoielab.c2b2.columbia.edu/lab/research/">Research</a>
              </li>
              <li>
              <a href="https://tavazoielab.c2b2.columbia.edu/lab/publications/">Publications</a>
              </li>
              <li class="current">
              <a href="https://tavazoielab.c2b2.columbia.edu/lab/tools/">Tools</a>
              </li>
              <li>
              <a href="https://tavazoielab.c2b2.columbia.edu/lab/personnel/">Personnel</a>
              </li>
              <li>
              <a href="https://tavazoielab.c2b2.columbia.edu/lab/pi-saeed/">PI: Saeed</a>
              </li>
              <li>
<a href="https://tavazoielab.c2b2.columbia.edu/lab/contact/">Contact</a>
              </li>
            </ul>
          </div>
          <div id="content">


            <!--
 <div id="main" class="normal">
   -->

   <h1>TEISER</h1>
   <h3 class="index">Supplementary web site for "Systematic discovery of structural elements governing stability of mammalian messenger RNAs"</h3>
   <h4 class="index">Hani Goodarzi, Hamed S. Najafabadi, Panos Oikonomou, Todd M. Greco, Lisa Fish, Reza Salavati, Ileana M. Cristea and Saeed Tavazoie</h4>
   <p>To contact us: {hgoodarzi@rockefeller.edu, hamed.najafabadi@utoronto.ca, st2744@columbia.edu}</p>
   
   <p>
  <a href="http://www.nature.com/nature/journal/v485/n7397/full/nature11013.html">LINK</a> <a href="/lab/publications/Goodarzi_et_al_Nature_2012.pdf">PDF</a>.<br />
   </p>

   <h2 class="index">How can I use TEISER?</h2>
   <div id="igetbox"><h4><a href="https://iget.c2b2.columbia.edu/"><abbr title="Integrated Genomics Exploration Tools">IGET</abbr>: Use <abbr title="Tool for Eliciting Informative Structural Elements in RNA">TEISER</abbr> online</a></h4><p>A limited version of <abbr title="Tool for Eliciting Informative Structural Elements in RNA">TEISER</abbr> is available as part of <a href="https://iget.c2b2.columbia.edu/"><abbr title="Integrated Genomics Explor\ation Tools">IGET</abbr></a>,<!-- the Integrated Genomics Exploration Tools website,--> an integrated platform for exploring large-scale gene expression and protein behavior dynamics. However, we strongly recommend downloading and using TEISER in house as it requires access to multiple nodes and computational power to complete the jobs in a reasonable time.</p>
<p> Use <a href="https://iget.c2b2.columbia.edu/"><abbr title="Integrated Genomics Exploration Tools">IGET</abbr></a> to discover motifs, pathways, and interactions.
</p>
</div>
	<p>
	TEISER can be <a href="http://magnet.c2b2.columbia.edu/?q=node/44&software=TEISER">downloaded</a> and used as a command line program (on a Unix or Cygwin machine).
	</p>

   <p>
   TEISER comes with pre-packaged sequence data for a number of organisms:<br />
   human (H. sapiens; 5' &amp; 3' UTRs),<br/>
   mouse (M. musculus; 5' &amp; 3' UTRs),<br/>
   yeast (S. cerevisiae; upstream &amp; downstream sequences)<br/>
   <!--#include virtual="species_insert.html"--></p>
<p>
 If your favorite organism is not in the list and you would like us to add it, please <a href="mailto:goodarzi@princeton.edu">contact us</a>. You can also add your own organisms (see tutorial or contact us for more details).  </p>
   <p>
   TEISER (for Tool for Eliciting Informative Structural Elements in RNA) is a robust and powerful framework for discovering post-transcriptional regulatory elemenets. <a href="tutorial.html">Click here for a tutorial on how to install and use TEISER</a>.</p>
 
   <h2 class="index">Supplemental results</h2>
   <h3 class="index">Discovering post-transcriptional <em>cis</em>-regulatory elements underlying mRNA stability</h3>
   <h4 class="index">Input data:</h4>
   <p>mRNA stability measurements: <a href="results/average_normalized_decay_rates.txt">.txt</a>  <a href="results/average_normalized_decay_rates.xls">.xls</a></p>
   <h4 class="index">Results:</h4>
   <p>False-discovery rate set at 0.0: <a href="results/teiser_rna_stability.pdf">.pdf</a>  <a href="results/teiser_rna_stability.txt">.txt</a></p>
   <p>False-discovery rate set at 0.1: <a href="results/teiser_rna_stability_long.pdf">.pdf</a>  <a href="results/teiser_rna_stability_long.txt">.txt</a></p>
   <p><br />Predicted target sites as weight-matrices: <a href="results/sRSM_WM.pdf">.pdf</a></p>

	<h3 class="index">Non-discovery analysis of experimental results</h3>
    <h4 class="index">Decoy vs. scrambled oligos: sRSM1</h4>
    <p>Expression profiles of decoy-transfected vs. scrambled transfected MDA-MB-231 cells: <a href="results/motif_0_dn_decoy_vs_scrambled_1.parsed.txt">Set1</a> <a href="results/motif_0_dn_decoy_vs_scrambled_2.parsed.txt">Set2</a></p>
    <p>TEISER results (non-discovery mode): <a href="results/motif_0_dn_decoy_vs_scrambled_1.parsed.pdf">Set1</a> <a href="results/motif_0_dn_decoy_vs_scrambled_2.parsed.pdf">Set2</a></p>
    <h4 class="index">HNRPA2B1 siRNA knock-down experiments</h4>
    <p>Expression profiles of siRNA-transfected vs. mock-transfected MDA-MB-231 cells: <a href="results/HNRPA2B1_KD_1.txt">siRNA1</a> <a href="results/HNRPA2B1_KD_2.txt">siRNA2</a> <a href="results/HNRPA2B1_KD_3.txt">siRNA3</a></p>
    <p>TEISER results (non-discovery mode): <a href="results/HNRPA2B1_KD_1.pdf">siRNA1</a> <a href="results/HNRPA2B1_KD_2.pdf">siRNA2</a> <a href="results/HNRPA2B1_KD_3.pdf">siRNA3</a></p>
    <p>Relative decay rates of siRNA-transfected vs. mock-transfected MDA-MB-231 cells: <a href="results/HNRNPA2B1_KD_decayrate.txt">siRNA pool</a></p>
    <p>TEISER results (non-discovery mode): <a href="results/HNRNPA2B1_KD_decayrate.pdf">siRNA pool</a></p>
    <h4 class="index">HNRPA2B1 RIP-chip</h4>
    <p>Transcript abundances in HNRPA2B1 immunoprecipitation samples: <a href="results/HNRPA2B1_CLIP_1.txt">Replicate1</a> <a href="results/HNRPA2B1_CLIP_2.txt">Replicate2</a></p>
    <p>TEISER results (non-discovery mode): <a href="results/HNRPA2B1_CLIP_1.pdf">Replicate1</a> <a href="results/HNRPA2B1_CLIP_2.pdf">Replicate2</a></p>
    <h4 class="index">HNRPA2B1 HITS-CLIP</h4>
    <p>Sequences bound by HNRPA2B1 in vivo: <a href="results/HNRPA2B1_footprint_1.txt">Replicate1</a> <a href="results/HNRPA2B1_footprint_2.txt">Replicate2</a></p>
    <p>TEISER results (non-discovery mode): <a href="results/HNRPA2B1_footprint_1.pdf">Replicate1</a> <a href="results/HNRPA2B1_footprint_2.pdf">Replicate2</a></p>
    
    <h4 class="index">CIMS analysis of HNRPA2B1 HITS-CLIP</h4>
    <p>Sequences bound by HNRPA2B1 in vivo: <a href="results/HNRNPA2.cims_teiser.fa">.fasta</a>    
    <p>TEISER results (non-discovery mode): <a href="results/HNRNPA2.cims_teiser.pdf">.pdf</a>
    <p>Examples of crosslinked bindign sites: <a href="results/HNRNPA2.cims_teiser_samples.pdf">.pdf</a>

    <h4 class="index">HNRPA2B1 data from Huelga et al. Cell (2012)</h4>
    <p>HNRPA2B1 siRNA knock-down: <a href="results/HNRNPA2B1_Huelga.txt">data</a> <a href="results/HNRNPA2B1_Huelga.pdf">result</a></p>
    <p>HNRPA2B1 HITS-CLIP: <a href="results/HNRPA2B1_CLIP_Huelga.txt">Binding sites (real and random)</a> <a href="results/HNRPA2B1_CLIP_Huelga.pdf">result</a></p>
    
   <h3 class="index">Deciphering the post-transcriptional regulatory program underlying mRNA stability</h3>
   <h4 class="index"><a href="/FIRE/">FIRE</a>: Finding linear motifs in addition to TEISER's structural elements</h4>
   <p>FIRE results: <a href="results/fire_rna_stability.pdf">.pdf</a>  <a href="results/fire_rna_stability.txt">.txt</a></p>
   <h4 class="index">Motif-motif interaction maps</h4>
   <p>Structural motifs vs. structural motifs (examplary interactions between sRSM0 and other elements): <a href="results/average_normalized_decay_rates.txt.mimatrix.short.pdf">.pdf</a>  <a href="results/average_normalized_decay_rates.txt.mimatrix.short.txt">.txt</a></p>
   <p>Structural motifs vs. linear motifs: <a href="results/motif_ssmotif.pdf">.pdf</a>  <a href="results/motif_ssmotif.txt">.txt</a></p>
   <h4 class="index"><a href="/iPAGE/">iPAGE</a>:Finding putative target pathways for the discovered elements</h4>
   <p>iPAGE results for structural motifs: <a href="results/average_normalized_decay_rates.txt.page.pdf">.pdf</a>  <a href="results/average_normalized_decay_rates.txt.page.txt">.txt</a></p>
   <p>iPAGE results for linear motifs: <a href="results/linear_motifs_page.pdf">.pdf</a>  <a href="results/linear_motifs_page.txt">.txt</a></p>
   <h4 class="index">Post-transcriptional Regulatory Program</h4>
   <p>All discovered dependencies are combined into a single <a href="results/regulatory_program.pdf">figure</a>.</p>
   
    
 </div>

</div>
 

<div class="clear">
              </div>
                                  <p><a href="http://www.columbia.edu/content/copyright.html">© 2011 Columbia University</a></p>



                                              </body></html>


